app/javascript/vue/books-data.js

```diff
@@ -1,7 +1,27 @@
 import axios from 'axios'
+import _ from 'lodash'
 
-export default new Promise((resolve, reject) => {
-  axios('/api/books')
-    .then((response) => resolve(response.data))
-    .catch((e) => reject(e))
-})
+function getAll () {
+  return new Promise((resolve, reject) => {
+    axios('/api/books')
+      .then((response) => resolve(response.data))
+      .catch((e) => reject(e))
+  })
+}
+
+function getById (id) {
```

Comment:
You do something wrong here. You need to implement another api endpoint (show method) and call that instead of using `getAll` and `_.find(books, { id: id })`.

Reply:
Exactly. It's a workaround. Do you think it's the time to make @isabloCh to implement it?

Reply:
@resivalex I just don't like the idea to start with workaround when we can implement endpoint and make logic right. If you think it's better to do it in separate PR it's totally ok with. Just don't want us to forget this.

Reply:
@isabloCh will make a decision. @isabloCh, will you?

Reply:
I asked you how to do @resivalex this is a mockery of me

Reply:
You said: "We don't need any new endpoints. We already can get books"

OK. Let's do it in the next pull-request

Reply:
Hahaha @isabloCh I hope you will survive 😂